libs/sdk-ui-kit/src/Header/Header.tsx

```diff
@@ -75,6 +75,9 @@ export interface IAppHeaderProps {
     activeColor?: string;
 
     userName: string;
+
+    disableHelpDropdown?: boolean;
+    onHelpClicked?: boolean;
```

Comment:
Please use present tense: `onHelpClick` as is already used in other props here.

Reply:
You got it 😄